eglib/configure.ac

```diff
@@ -130,6 +130,10 @@ AM_CONDITIONAL(TARGET_WIN32, test x$OS = xWIN32)
 AM_CONDITIONAL(PLATFORM_DARWIN, test x$platform_darwin = xyes)
 AM_CONDITIONAL(PLATFORM_ANDROID, test x$platform_android = xyes)
 
+# Defined for all targets/platforms using classic Windows API support.
+AC_DEFINE(HAVE_CLASSIC_WINAPI_SUPPORT, 1, [Use classic Windows API support])
+AC_DEFINE(HAVE_UWP_WINAPI_SUPPORT, 0, [Don't use UWP Windows API support])
```

Comment:
Right now you can do this manually (by changing winconfig.h) and this is how it is currently tested. Windows builds have never used the result from configure steps, it uses the checked in version of winconfig.h. This file uses the winfamilyapi.h to set the right API family based on build targets and it is possible to manually override just to test mono API Surface.